resources/files/facter.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+unset LD_PRELOAD
+unset LD_LIBRARY_PATH
```

Comment:
Also if the whole point of changing the symlinks to bash scripts is to modify these variables, make it makes more sense to modify these variables in the actual project executables rather than create this extra layer of indirection?